tests/tcp.rs

```diff
@@ -571,7 +571,9 @@ fn connect_error() {
 
         for event in &events {
             if event.token() == Token(0) {
-                assert!(event.is_writable());
+		// With fastopen we would be able to write
+		// Without fastopen we would be getting the connection error
```

Comment:
Needs formatting:
```suggestion
		    // With fastopen we would be able to write.
		    // Without fastopen we would be getting the connection error.
```

Reply:
ah, right, my editor wasn't set up to show me the difference between spaces and tabs, now fixed